ONSdigital/eq-author
# Add and delete metadata

### What is the context of this PR?
Implements functionality required to add/modify/delete Metadata.

### How to review 
- Run tests
- Ensure feature works as https://trello.com/c/giG4PIKu/548-add-metadata-12 and https://trello.com/c/ZnCEpBZZ/557-delete-metadata-2
